### PR TITLE
Adding lambda version parameter to outputs.tf

### PR DESCRIPTION
**Pull request by hknerts**
Some projects need lambda version to associate it some resource

### CODE REVIEW

**Comment by deeptechs** [comment]
Hi @ivankatliarchuk ,

It would be great to use terraform v12 feature about exporting whole object in output.tf like:

```
# Configuration for Terraform 0.12

output "lambda" {
  value = aws_lambda_function.example
}
```

Source: 
https://www.hashicorp.com/blog/terraform-0-12-rich-value-types/#resources-and-modules-as-values

File: outputs.tf [hunk 1]
```diff
@@ -3,6 +3,11 @@ output "arn" {
   description = "Lambda ARN"
 }
 
+output "arn_version" {
```
Comment (ivankatliarchuk) [inline outputs.tf]: `version` is just fine

**Comment by hknerts** [comment]
i changed it but some checks are failing

**Review by ivankatliarchuk — APPROVED** [review]
I can do the rest myself in the next few days.
Basically, the `pre-commit-hook` need to be installed and someone should run a command `pre-commit run --all-files` this will validate/fix all the issues

**Pull request merged**